cardano-api/src/Cardano/Api/Byron.hs

```diff
@@ -68,9 +68,12 @@ module Cardano.Api.Byron
     NetworkMagic(..),
 
     -- ** Conversions
+    fromByronTxIn,
     toByronNetworkMagic,
     toByronProtocolMagicId,
     toByronRequiresNetworkMagic,
+
+    makeByronTransaction,
```

Comment:
Wasn't this already exported via `Cardano.Api`?

Reply:
It wasn't but I will move it there.